monitoring/messagesigning/run_well_known.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
This file should not be required anymore since it will be replaced by the new `mock_uss/run_locally_[SERVICE_NAME].sh`.